src/cmap/connection.ts

```diff
@@ -562,6 +562,15 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
       callback(err);
     }
   }
+
+  moreToComeCommand(
```

Comment:
Just out of curiosity, what was the need for this? Are there upcoming changes that will alter the behaviour of this method?

Reply:
No, this method will remain as is until we remove the old connection class entirely. This is so the monitor can be refactored to support either Connection or ModernConnection without needing to gaurd on which instance of the connection class it is using. I added the empty method to ModernConnection to make it a bit more clear, the next segment of work will fill out that method and make Monitors start testing with the new connection class.

Reply:
Ah ok, thanks for the explanation.